### PR TITLE
Icodara 243

### PR DESCRIPTION
**Pull request by samhPA**
Added Matt requests

### CODE REVIEW

File: src/pages/dashboard/DataAccessRequests/csvUtils.js [hunk 1]
```diff
@@ -145,11 +145,13 @@ const buildOneRow = (dar, applicant) => {
 	const row = [
 			dar._id,
 			dar.applicationStatus,
+			dar.applicationStatusDesc,
```
Comment (samhPA) [inline src/pages/dashboard/DataAccessRequests/csvUtils.js:148]: Adding "Status notes" to the csv file

File: src/pages/dashboard/DataAccessRequests/csvUtils.js [hunk 2]
```diff
@@ -145,11 +145,13 @@ const buildOneRow = (dar, applicant) => {
...
 			dar.createdAt.substring(0, 10), // YYYY-MM-DD
 			dar.updatedAt.substring(0, 10), // YYYY-MM-DD
-			dar.datasetTitles[0],
```
Comment (samhPA) [inline src/pages/dashboard/DataAccessRequests/csvUtils.js:150]: csv now present multi datasets (if there are any), separated by ";"

File: src/pages/dashboard/DataAccessRequests/csvUtils.js [hunk 3]
```diff
@@ -145,11 +145,13 @@ const buildOneRow = (dar, applicant) => {
...
 			dar._id,
 			dar.applicationStatus,
+			dar.applicationStatusDesc,
 			dar.createdAt.substring(0, 10), // YYYY-MM-DD
 			dar.updatedAt.substring(0, 10), // YYYY-MM-DD
-			dar.datasetTitles[0],			
+			dar.datasetTitles.join(';'),			
 
-			applicant.safeprojectapplicantfullname, //"Project submitted by"
+			dar.aboutApplication.projectName, // Project name
```
Comment (samhPA) [inline src/pages/dashboard/DataAccessRequests/csvUtils.js:153]: Added project name to the csv as Matt requested

File: src/pages/dashboard/DataAccessRequests/csvUtils.js [hunk 4]
```diff
@@ -180,9 +182,11 @@ const buildHeaders = () => {
 	return [
 				"ID", 
 				"Status", 
-				"Created At", 
+				"Status Notes", 
+				"Created At",
 				"Updated At", 
-				"Datasets", 
+				"Datasets",
+				"Project name",
```
Comment (samhPA) [inline src/pages/dashboard/DataAccessRequests/csvUtils.js:189]: Adding the headers accordingly

**Review by PaulMcCaffertyPA — APPROVED** [review]

**Pull request merged**